auth_proxy/views/oauth/views.py

```diff
@@ -82,18 +83,27 @@ def debug_create_token(*args, **kwargs):
     return jsonify({'access_token': token.access_token, 'refresh_token': token.refresh_token})
 
 
-@BP.route('/debug/introspect/<token>', methods=['GET'])
-def debug_token_introspection(token, *args, **kwargs):
+@BP.route('/debug/introspect', methods=['GET'])
```

Comment:
Thanks, and sorry for all the back and forth on this one. To match the OAuth 2.0 Bearer token specification, can you call this parameter `access_token`? (https://tools.ietf.org/html/rfc6750#section-2.3)